### PR TITLE
feat(lib): Cross-platform compatability

### PR DESCRIPTION
**Pull request by BenBeattieHood**
I needed to run your superb npm package on Windows, so would like to propose these small changes to allow this please.

### CODE REVIEW

**Comment by BenBeattieHood** [comment]
(cc @vvo)

**Comment by vvo** [comment]
@BenBeattieHood Thanks! Do you have experience in testing CDK constructs? Only asking because a good way to ensure your work and mine will not break in the future would be to setup an example CDK construct and build it on windows/linux on GitHub actions.

File: src/NodejsFunction.ts [hunk 1]
```diff
@@ -123,15 +123,17 @@ export class NodejsFunction extends lambda.Function {
       };
     }, {});
 
+    const nodeifyPath = (path: string) => path.replace(/\\/g, '\\\\');
```
Comment (vvo) [inline src/NodejsFunction.ts]: Can you explain why this is needed + add a comment maybe? Thanks!

**Comment by BenBeattieHood** [comment]
> @BenBeattieHood Thanks! Do you have experience in testing CDK constructs? Only asking because a good way to ensure your work and mine will not break in the future would be to setup an example CDK construct and build it on windows/linux on GitHub actions.

Only a little, and only a little on GitHub actions too. This might change next year, but sorry not to be of much use right now.

File: src/NodejsFunction.ts (hunk 1 above)
Comment (BenBeattieHood) [reply src/NodejsFunction.ts]: Done 👍 good call out.

**Pull request merged**